test/integration/puppetlabs/services/legacy_routes/legacy_routes_test.clj

```diff
@@ -6,7 +6,17 @@
             [schema.test :as schema-test]
             [puppetlabs.services.jruby.jruby-testutils :as jruby-testutils]
             [me.raynes.fs :as fs]
-            [puppetlabs.trapperkeeper.testutils.logging :as logutils]))
+            [puppetlabs.trapperkeeper.testutils.logging :as logutils]
+            [puppetlabs.services.request-handler.request-handler-service :as handler]
+            [puppetlabs.services.jruby.jruby-puppet-service :as jruby]
+            [puppetlabs.services.puppet-profiler.puppet-profiler-service :as profiler]
+            [puppetlabs.trapperkeeper.services.webserver.jetty9-service :as webserver]
+            [puppetlabs.trapperkeeper.services.webrouting.webrouting-service :as webrouting]
+            [puppetlabs.services.config.puppet-server-config-service :as ps-config]
+            [puppetlabs.services.legacy-routes.legacy-routes-service :as legacy-routes]
+            [puppetlabs.services.puppet-admin.puppet-admin-service :as admin]
+            [puppetlabs.services.ca.certificate-authority-disabled-service :as disabled-ca]
+            [puppetlabs.kitchensink.core :as ks]))
```

Comment:
Ahh yes. I didn't realize that all my Clojure inspections were turned off in IntelliJ.